doc/nrf/known_issues.rst

```diff
@@ -1185,6 +1185,20 @@ NCSDK-13923: Device may crash during Bluetooth bonding
 
   **Workaround:** Manually cherry-pick and apply the commit with the fix from the ``main`` branch (commit hash: ``4870fcd8316bd3a4b53ca0054f0ce35e1a8c567d``).
```

Comment:
Fix commit message: double space before `machine` and `:` instead of `.` at the end of the sentence.